deploy-os_servers.yml

```diff
@@ -486,6 +592,55 @@
         wait: true
         timeout: "{{ openstack_api_timeout }}"
 ##############################################################################
+# Configure IRODS servers from inventory using Openstack API.
+##############################################################################
+- name: Create IRODS catalogus servers.
+  hosts: irods
+  connection: local
+  gather_facts: false
+  vars:
+    #
+    # Disable Ansible's interpretor detection logic,
+    # which fails to use the interpretor from an activated virtual environment
+    # and hence fails to find the OpenStackSDK if it was installed in a virtual environment.
+    #
+    - ansible_python_interpreter: python
+  tasks:
+    - name: Create persistent data volume for irods server.
+      openstack.cloud.volume:
+        display_name: "{{ inventory_hostname }}-volume"
+        size: "{{ local_volume_size_repo }}"
```

Comment:
`local_volume_size_repo` is confusing as it suggests it is for the Pulp repo server. We should either use a default volume size her or create a `local_volume_size_irods` variable.